services/ui-src/src/components/fields/RadioField.tsx

```diff
@@ -5,7 +5,6 @@ import { ChoiceTemplate, PageElement, RadioTemplate } from "types";
 import { parseHtml, useStore } from "utils";
 import { ChoiceList as CmsdsChoiceList } from "@cmsgov/design-system";
 import { Page } from "components/report/Page";
-import { ChoiceProps } from "@cmsgov/design-system/dist/react-components/types/ChoiceList/ChoiceList";
```

Comment:
I could not find an appropriate type in the latest `@cmsgov/design-system`.  The checks are happy without it.